### PR TITLE
[wasm] Fix AOT.

### PR DESCRIPTION
**Pull request by vargaz**

### CODE REVIEW

File: sdks/builds/wasm.mk [hunk 1]
```diff
@@ -88,17 +88,19 @@ _wasm-$(1)_CONFIGURE_FLAGS= \
...
 
 $$(eval $$(call CrossRuntimeTemplate,wasm-$(1),$$(if $$(filter $$(UNAME),Darwin),$(2)-apple-darwin10,$$(if $$(filter $$(UNAME),Linux),$(2)-linux-gnu,$$(error "Unknown UNAME='$$(UNAME)'"))),$(3)-unknown-none,$(4),$(5),$(6)))
 
+ifdef ENABLE_WASM_CROSS
 wasm_TARGETS += wasm-$(1)-$$(CONFIGURATION) $(5)
+endif
 
 endef
 
-ifeq ($(DISABLE_WASM_CROSS),)
```
Comment (kumpera) [inline sdks/builds/wasm.mk]: Won’t that force users with cross disabled to provision mxe?

Comment (vargaz) [reply sdks/builds/wasm.mk]: Don't think so, this just enables the make targets.

**Comment by lewurm** [comment]
@monojenkins squash

**Comment by kjpou1** [comment]
Tested this and it fixes: `WASM-ERR: Invalid function pointer called with signature 'iii'.`

**Comment by lewurm** [comment]
@monojenkins build failed

**Comment by vargaz** [comment]
@monojenkins build failed

**Comment by vargaz** [comment]
@monojenkins build failed

**Pull request merged**